### ISSUE TITLE
ALOpsAppPublish does not report errors

### ISSUE THREAD

**Issue by gintuoklis** [HodorNV/ALOps#82]
Hey Guys, 

I noticed that some of my extensions goes through deploy pipeline (everything is 'Green'). But, extension is not installed on the server - just published. 

If I try to run installation manual using Install-NavApp i find actual issue. 

- In the first case, it was identical object number within another extension.
`Install-NAVApp : The object name 'Very Serious Object' can only appear once for the type 'CodeUnit'.`

-  In second case, it was caused by same field ID used across multiple extensions.
`Install-NAVApp : An item with the same key has already been added.`

My deployment tasks is super simple (thanks to ALOps ;))
```
              - task: ALOpsAppPublish@1
                displayName: 'ALOps App Publish'
                inputs:
                  nav_serverinstance: NAV
```

Could you please fix this issue, and show errors from Install-Navapp. Now you think that everything is fine, but extension is not actually deployed. 

Thank You.

**Comment by waldo1001** [comment]
Wow, that would be bad.  Thanks for taking the time - will investigate!

**Comment by waldo1001** [comment]
Hi @gintuoklis , can you confirm this has been solved?

**Comment by gintuoklis** [comment]
you fixed it as part of #122, so yes it was solved
